package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
What was the problem with using the `lib/index` here? It seems like dist builds in `object-assign` and `error-stack-parser` right? Do these not work well as separate dependencies?